analytics.go

```diff
@@ -94,3 +151,45 @@ func (c *Client) getAggregatedData(ctx context.Context, analytics AnalyticsReque
 
 	return analyticsResponse, nil
 }
+
+// GetRawDataSingleIncident gets the raw analytics for the requested incident.
+func (c *Client) GetRawDataSingleIncident(ctx context.Context, id string) (RawData, error) {
+	h := map[string]string{
+		"X-EARLY-ACCESS": "analytics-v2",
```

Comment:
They probably were at the time you put up the PR but these APIs are no longer "Early Access"